src/pages/NewsList/NewsList.tsx

```diff
@@ -89,7 +92,12 @@ export default function NewsList() {
 			{newsData ? (
 				<div className="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 gap-5">
 					{newsData.map((news: NewsListProps) => (
-						<div key={news.title} className="flex flex-col h-full p-6 mb-4 bg-gray-50 rounded-lg shadow-lg">
+						<div key={news.title} className="flex flex-col h-full p-6 mb-4 bg-gray-50 rounded-2xl shadow-lg">
+							{news.banner_image ? (
+								<img src={news.banner_image} alt={news.title} className="mb-4 rounded-xl shadow-lg" />
+							) : (
+								<img src={placeholder} alt="article" className="mb-4 rounded-xl shadow-lg" />
```

Comment:
In a normal app, I would probably want this to be a component with this src and alt as defaults. Also, I think the alt could be better (either better text or empty), but I understand it is not a focus now :)